src/coreclr/src/jit/emitarm64.cpp

```diff
@@ -3218,6 +3229,53 @@ emitter::code_t emitter::emitInsCode(instruction ins, insFormat fmt)
     return false;
 }
 
+//------------------------------------------------------------------------
+// insGetLoadStoreVectorSelem: Returns a number of structure elements for a Load/Store Vector instruction
+//
+/*static*/ unsigned emitter::insGetLoadStoreVectorSelem(instruction ins)
+{
+    unsigned selem = 0;
```

Comment:
could we call this `elementCount` or `structureElements` or something a bit more descriptive?

Reply:
(same goes for the method name)

Reply:
sure, I can rename it to `insGetLoadStoreVectorStructureElements`

Reply:
After more carefull reading Arm specs I don't think it should be called `insGetLoadStoreVectorStructureElements`

It should be `insGetLoadStoreRegisterListSize`
For LD1 (2 registers) - number of structure elements = 1 since the instruction loads 2 single-element structures to 2 registers. 

And this function reflects how many registers `ins` operates on

Reply:
That's sounds good. I just don't think `Selem` is very self explanatory.